Source/EventHorizon/Consumer/EventProcessor.cs

```diff
@@ -49,20 +54,18 @@ public class EventProcessor : IEventProcessor
         public EventProcessorId Identifier { get; }
 
         /// <inheritdoc/>
-        public async Task<IProcessingResult> Process(CommittedEvent @event, PartitionId partitionId, CancellationToken cancellationToken)
-        {
-            _logger.Trace($"Processing event '{@event.Type.Id}' in scope '{Scope}' from microservice '{_subscriptionId.ProducerMicroserviceId}' and tenant '{_subscriptionId.ProducerTenantId}'");
-
-            await _receivedEventsWriter.Write(@event, _consentId, Scope, cancellationToken).ConfigureAwait(false);
-            return new SuccessfulProcessing();
-        }
+        public Task<IProcessingResult> Process(CommittedEvent @event, PartitionId partitionId, CancellationToken cancellationToken) => Process(@event, cancellationToken);
 
         /// <inheritdoc/>
-        public async Task<IProcessingResult> Process(CommittedEvent @event, PartitionId partitionId, string failureReason, uint retryCount, CancellationToken cancellationToken)
+        public Task<IProcessingResult> Process(CommittedEvent @event, PartitionId partitionId, string failureReason, uint retryCount, CancellationToken cancellationToken) => Process(@event, cancellationToken);
```

Comment:
Could we maybe have some logs here writing a trace that it is retrying before calling `Process`?

Reply:
https://github.com/dolittle-runtime/Runtime/pull/408/commits/4b047c18a0f2698b6f79b77036442c4daee75134